src/app.module.ts

```diff
@@ -21,6 +21,8 @@ import { UdiseService } from 'src/services/udise/udise.service';
 import { TelemetryService } from './services/telemetry/telemetry.service';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { UsersModule } from './services/users/users.module';
+import { ClaimAttestService } from './claimAttest/claimAttest.service';
```

Comment:
@YoginiTayade remove service claimattestservice import in app.module only keep claimattestmodule in app. module

Reply:
resolved